modules/acquia_cms_headless/modules/acquia_cms_headless_ui/acquia_cms_headless_ui.module

```diff
@@ -96,6 +97,21 @@ function acquia_cms_headless_ui_form_node_form_alter(array &$form, FormStateInte
     ];
   }
   $form['#process'][] = 'acquia_cms_headless_ui_process_node_type_workflow_options';
+  $form['actions']['submit']['#submit'][] = 'acquia_cms_headless_ui_redirect_user_to_site_preview';
```

Comment:
@panshulK I think the redirection of form is handling from function added above i.e 
`Redirect::entityForm($form, $form_state);`
and looks like handled from [here](https://github.com/acquia/acquia_cms/blob/develop/modules/acquia_cms_headless/modules/acquia_cms_headless_ui/src/Redirect.php#L68).
Please see.

Reply:
Hey @vishalkhode1  I've updated the PR with changes discussed please review.